gusto/configuration.py

```diff
@@ -48,6 +48,8 @@ class OutputParameters(Configuration):
     steady_state_error_fields = []
     #: List of fields for computing perturbations
     perturbation_fields = []
+    #: Dictionary of fields and points at which to dump them
+    point_data = {}
```

Comment:
You should describe the format of these points.  It's not the format the `Function.at` wants (which is a `(N, geometric_dimension)` shape array of points).  Is there a reason you didn't use that format?